Makefile

```diff
@@ -15,6 +15,12 @@ REVIEWDOG_REPO=github.com/haya14busa/reviewdog/cmd/reviewdog
 
 # Runs complete testsuites (unit, system, integration) for all beats with coverage and race detection.
 # Also it builds the docs and the generators
+
+.PHONY: init
+init:
```

Comment:
can we call this `setup-commit-hook` or something like this? Otherwise I think this gets confused with `init` for the `beats` project which I don't think it is.

Reply:
sure! 
I also don't mean to enforce this on the beats team, just a suggestion.

Reply:
I think it's great to have this in they way it is as it makes it optional. I see it especially useful for people new to the project. Instead of having to tell them all the commands to run, they only need to run one to setup the pre-commit hook.